src/cls/rgw_gc/cls_rgw_gc.cc

```diff
@@ -369,7 +369,9 @@ static int cls_rgw_gc_queue_remove_entries(cls_method_context_t hctx, bufferlist
   }
 
   //Update urgent data map
+  head.bl_urgent_data.clear();
```

Comment:
I see now;  I guess having a buffer::list in the head itself, rather than on the stack for some operation or other made this a bit more likely to show up;  the latter is the more common pattern in the ceph code